### PR TITLE
Add full commands for supported configs

### PR DESCRIPTION
**Pull request by kdamaszk**
# What does this PR do?

Adds full commands for example configs for easier reproduction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,13 +112,129 @@ Additional hints to quantize model for TGI when using `run_lm_eval.py`:
...
+   -e HABANA_VISIBLE_DEVICES=all \
+   -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+   -e HF_HUB_ENABLE_HF_TRANSFER=1 \
+   -e HUGGING_FACE_HUB_TOKEN=$hf_token \
+   -e PREFILL_BATCH_BUCKET_SIZE=1 \
+   -e BATCH_BUCKET_SIZE=16 \
+   -e PAD_SEQUENCE_TO_MULTIPLE_OF=128 \
+   --cap-add=sys_nice \
+   --ipc=host \
+   ghcr.io/huggingface/tgi-gaudi:2.0.0 \
```
Comment (sureshnam) [inline README.md:134]: Could you add dock as variable to remind users when using to change according to the latest tag /available docker or we should update it for every release tag

File: README.md [hunk 2]
```diff
@@ -112,13 +112,129 @@ Additional hints to quantize model for TGI when using `run_lm_eval.py`:
...
+   -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+   -e HF_HUB_ENABLE_HF_TRANSFER=1 \
+   -e HUGGING_FACE_HUB_TOKEN=$hf_token \
+   -e PREFILL_BATCH_BUCKET_SIZE=1 \
+   -e BATCH_BUCKET_SIZE=64 \
+   -e PAD_SEQUENCE_TO_MULTIPLE_OF=128 \
+   -e QUANT_CONFIG=./quantization_config/maxabs_quant.json \
+   --cap-add=sys_nice \
+   --ipc=host \
+   ghcr.io/huggingface/tgi-gaudi:2.0.0 \
```
Comment (sureshnam) [inline README.md:164]: Could you add dock as variable to remind users when using to change according to the latest tag /available docker or we should update it for every release tag

File: README.md [hunk 3]
```diff
@@ -112,13 +112,129 @@ Additional hints to quantize model for TGI when using `run_lm_eval.py`:
...
+   -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+   -e HF_HUB_ENABLE_HF_TRANSFER=1 \
+   -e HUGGING_FACE_HUB_TOKEN=$hf_token \
+   -e PT_HPU_ENABLE_LAZY_COLLECTIVES=true \
+   -e PREFILL_BATCH_BUCKET_SIZE=1 \
+   -e BATCH_BUCKET_SIZE=256 \
+   -e PAD_SEQUENCE_TO_MULTIPLE_OF=128 \
+   --cap-add=sys_nice \
+   --ipc=host \
+   ghcr.io/huggingface/tgi-gaudi:2.0.0 \
```
Comment (sureshnam) [inline README.md:192]: Could you add dock as variable to remind users when using to change according to the latest tag /available docker or we should update it for every release tag

File: README.md [hunk 4]
```diff
@@ -112,13 +112,129 @@ Additional hints to quantize model for TGI when using `run_lm_eval.py`:
...
+   -e HF_HUB_ENABLE_HF_TRANSFER=1 \
+   -e HUGGING_FACE_HUB_TOKEN=$hf_token \
+   -e PT_HPU_ENABLE_LAZY_COLLECTIVES=true \
+   -e PREFILL_BATCH_BUCKET_SIZE=1 \
+   -e BATCH_BUCKET_SIZE=512 \
+   -e PAD_SEQUENCE_TO_MULTIPLE_OF=128 \
+   -e QUANT_CONFIG=./quantization_config/maxabs_quant.json \
+   --cap-add=sys_nice \
+   --ipc=host \
+   ghcr.io/huggingface/tgi-gaudi:2.0.0 \
```
Comment (sureshnam) [inline README.md:226]: Could you add dock as variable to remind users when using to change according to the latest tag /available docker or we should update it for every release tag

File: README.md (hunk 1 above)
Comment (kdamaszk) [reply README.md:134]: After each release we are updating the README with the new docker image, so it can stay in current form

**Comment by kdamaszk** [comment]
@regisss please review

**Review by regisss — APPROVED** [review]

**Pull request merged**